src/pudl/etl/denormalized_assets.py

```diff
@@ -1,4 +1,13 @@
-"""Output table assets."""
+"""Denormalized tables, based on other simple normalized tables.
+
+Right now these will mostly be tables defined in `pudl.output`. Ultimately it would
+also include any tables built by joining and aggregating normalized and analysis
+tables. These are probably data warehouse style tables with a lot of duplicated
+information, ready for analysts to use.
```

Comment:
Right now we have `pudl.etl.output_assets` and `pudl.etl.analysis_assets`. I was thinking these modules would be place holders until we start converting output and analysis tables. Might be easier to just wrap the functions in `pudl.output` instead of moving things into these place holder modules.  (Not a requested change or blocker for this PR just a thought.)

Reply:
I just wanted to conceptually separate "outputs" (which we get fuzzy with) into the denormalized vs. analysis categories from the beginning. Agree that for now we're probably going to just define these assets over wherever the existing functions are, and we can give them explicit asset groups over there.

Reply:
After the transition to a more data-warehouse style architecture, I think that they day-to-day use is mostly going to be using the denormalized tables (as we do now, via `PudlTabl`) so maybe those are the tables that end up not having a funny prefix?

Reply:
Sounds good to me! I like the separation of analysis and output for now.

You're suggesting adding a funny prefix to the normalized tables to differentiate them from the denormalized tables because most folks will be interacting with the denormed ones?

Reply:
I'm imagining something like `raw_generation_eia923`, `clean_generation_eia923`, `norm_generation_eia923` (or other meaningful indicators of what step you're at) and then just have `generation_eia923` for the easy-to-use denormalized table.